web/src/components/atoms/ProTable/index.tsx

```diff
@@ -1,5 +1,13 @@
-import ProTable from "@ant-design/pro-table";
-import type { ProTableProps } from "@ant-design/pro-table";
+import ProTable, { ListToolBarProps } from "@ant-design/pro-table";
+import type { ProTableProps, ProColumns } from "@ant-design/pro-table";
```

Comment:
Is there a reason ListToolBarProps isn't imported with ProTableProps and ProColumns